### ISSUE TITLE
Future plans ?

### ISSUE THREAD

**Issue by markdoerr** [pysemtec/pysemtec.org#3]
Dear Evgeny (@blokhin), 

while having a very similar idea of creating a forum for python focused semantic webtools [https://github.com/semanticpy]( https://github.com/semanticpy ), I found your github group.
It seems that the development of this webpage / forum / group  slowed down.
What are your future plans ? Are you considering developing own python tools for ontology development / reasoning / semantic web ?
What are your visions / perspectives (and time constraints) ?

Thanks, mark

**Comment by blokhin** [comment]
Dear Mark,

the idea of this GitHub org belongs to Carsten @cknoll. I believe, he will definitely like and support your effort, and so either do I. The OWL as a standard has a very complicated unlucky history, and, as a result, the modern robust open-source toolchain is not there. As a side project, I'm developing [pyfactxx](https://github.com/tilde-lab/pyfactxx), but it requires more time, and therefore the progress is very slow.

As said, I'm very much open to collaboration and ready to invest more efforts. Let's try to connect in an e-meeting e.g. at the first week of May 2023? Dear Carsten @cknoll, please, also join us and let us know your available time slots.

**Comment by markdoerr** [comment]
Thanks a lot, Evgeny @blokhin, 
for your fast reaction :)
I am looking forward to our meeting - also with Carsten @cknoll .
I will write to both of you an e-mail.  Cheers

**Comment by cknoll** [comment]
> It seems that the development of this webpage / forum / group slowed down.

Yes. This is probably my fault. I put some initial effort into this project but did not keep up this effort.

For all third party readers: there is an ongoing e-mail conversation which will probably result in an online meeting at the beginning of may. I am not sure yet how public this will be but I guess it will be announced in this thread.
